src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
+    .reduce((stylesObject, style) => {
+      const [key, value] = style.split(':').map((item) => item.trim());
+
+      if (key && value) {
+        stylesObject[key] = value;
```

Comment:
Checking for the presence of both key and value before assigning them to the stylesObject is a good practice to avoid adding invalid entries.